kythe/data/simple_vnames.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "pattern": "bazel-out/[^/]+/bin/external/[^/]+/(.+)",
+    "vname": {
+      "corpus": "CORPUS",
+      "path": "@1@"
```

Comment:
These files do not exist in the repository (and may have conflicting paths) so we need to put a `root` on them.  It looks like the next rule will cover this case and we can just delete this rule.

Reply:
Removed.